service/src/main/java/uk/gov/hmcts/reform/fpl/enums/CaseRole.java

```diff
@@ -6,21 +6,34 @@
     EPSMANAGING,
     LAMANAGING,
     SOLICITOR,
+    SOLICITOR_A("solicitorA"),
+    SOLICITOR_B("solicitorB"),
+    SOLICITOR_C("solicitorC"),
+    SOLICITOR_D("solicitorD"),
+    SOLICITOR_E("solicitorE"),
+    SOLICITOR_F("solicitorF"),
+    SOLICITOR_G("solicitorG"),
+    SOLICITOR_H("solicitorH"),
+    SOLICITOR_I("solicitorI"),
     LABARRISTER,
     BARRISTER,
     CAFCASSSOLICITOR;
 
-    private String formattedName;
+    private final String formattedName;
 
     CaseRole() {
         this.formattedName = String.format("[%s]", name());
     }
 
+    CaseRole(String formattedName) {
+        this.formattedName = formattedName;
```

Comment:
this is not ok, all roles should be formatted as [...] so e.g [solicitorA] and not solicitorA